core/src/test/java/org/elasticsearch/script/ScriptServiceTests.java

```diff
@@ -306,12 +249,11 @@ public void testFineGrainedSettings() throws IOException {
         }
 
         buildScriptService(builder.build());
-        createFileScripts("expression", "mustache", "dtest");
 
         for (ScriptType scriptType : ScriptType.values()) {
             //make sure file scripts have a different name than inline ones.
             //Otherwise they are always considered file ones as they can be found in the static cache.
-            String script = scriptType == ScriptType.FILE ? "file_script" : "script";
+            String script = "script";
             for (ScriptContext scriptContext : this.scriptContexts) {
                 //fallback mechanism: 1) engine specific settings 2) op based settings 3) source based settings
                 Boolean scriptEnabled = engineSettings.get(dangerousScriptEngine.getType() + "." + scriptType + "." + scriptContext.getKey());
```

Comment:
It won't let me review line 278 where there is still an assertion about file scripts.